reactDashboard/src/components/HeatMap/Heatmap.jsx

```diff
@@ -316,15 +313,13 @@ function Heatmap({
       <Table size="sm">
         <thead>
           <tr>
-            <th>
-              <div>{areaTitle()}</div>
-              <div className="subheading">{areaSubTitle()}</div>
+            <th className="heatmap-table-heading">
```

Comment:
The class name is redundant here - you can delete it

Reply:
deleted